web/src/store/reducers/jobs.ts

```diff
@@ -49,6 +67,30 @@ export default (state = initialState, action: IJobsAction): IJobsState => {
       return { ...state, result: state.result.filter((e) => e.name !== payload.jobName) }
     case DELETE_JOB_SUCCESS:
       return { ...state, deletedJobName: payload.jobName }
+    case FETCH_JOB_TAGS:
+      return { ...state, isLoading: true }
+    case FETCH_JOB_TAGS_SUCCESS:
+      return {
+        ...state,
+        isLoading: false,
+        jobTags: payload.jobTags,
+      }
+    case ADD_JOB_TAG:
+      return {
+        ...state,
+      }
+    case ADD_JOB_TAG_SUCCESS:
+      return {
+        ...state,
+      }
+    case DELETE_JOB_TAG:
+      return {
+        ...state,
+      }
+    case DELETE_JOB_TAG_SUCCESS:
+      return {
+        ...state,
+      }
```

Comment:
Do we need any of these, if they just reflect the same state?

Reply:
Yip happy to remove.